upgrade/1.2/README.md

```diff
@@ -27,6 +27,10 @@ mentioned explicitly on this page, see [resource material](resource_material/REA
     For more information about modifying `customizations.yaml` and tuning for specific systems, see
     [Post Install Customizations](../../operations/CSM_product_management/Post_Install_Customizations.md).
 
+## Known issues
+
+- `kdump` (kernel dump) may hang and fail on NCNs in CSM 1.2 (Raspberry).
```

Comment:
```suggestion
- `kdump` (kernel dump) may hang and fail on NCNs in CSM 1.2 (HPE Cray EX System Software 22.07 release).
```
We shouldn't be using the codename in the public documentation.

Reply:
This should also be fixed in the 1.2 version that was merged. I can open a new PR for this change if needed.

Reply:
Please and thank you.